pages/config-service-configuration/add-or-edit-chain.mdx

```diff
@@ -0,0 +1,195 @@
+
+### `Description`
+
+A brief description of the chain.
+
+### `Chain logo URI`
+
+The URI of the chain's logo. This will be displayed in the UI.
+
+### `L2`
+
+Whether this chain is a Layer 2 chain. This has important consequences on the indexing ability of the chain. If this chain is an Ethereum Layer 2, not checking this box will increase the indexing time, hence possibly missing the correct indexing of some transactions.
+
+### `Is testnet`
+
+Whether this chain is a testnet. This will be indicated in the UI.
+
+### `RPC authentication`
+
+Whether the RPC endpoint requires authentication to access its data. If `API key path` is set, the API key will be read from the RPC URI.
+
+### `Rpc uri`
+
+The URI of the chain's RPC endpoint. This is the endpoint where the Config Service will fetch the chain's data.
+
+### `Safe Apps RPC authentication`
+
+Whether the RPC endpoint dedicated to Safe Apps will require authentication. If `API key path` is set, the API key will be read from the Safe Apps RPC URI.
+
+### `Safe Apps rpc uri`
+
+The URI of the chain's RPC endpoint dedicated to Safe Apps.
+
+### `Public RPC authentication`
+
+Whether the fallback public RPC endpoint will require authentication. If `API key path` is set, the API key will be read from the Public RPC URI.
+
+### `Public rpc uri`
+
+The URI of the chain's RPC public endpoint.
+
+### `Block explorer uri address template`
+
+The URI template for the chain's block explorer addresses. This will be used to generate address links to the block explorer. For example: `https://etherscan.io/address/{{address}}`.
+
+### `Block explorer uri tx hash template`
+
+The URI template for the chain's block explorer transaction hashes. This will be used to generate `txHash` links to the block explorer. For example: `https://etherscan.io/tx/{{txHash}}`.
+
+### `Block explorer uri api template`
+
+The URI template for the chain's block explorer API. This will be used to fetch data from the block explorer. For example: `https://api.etherscan.io/api`.
+
+### `Beacon chain explorer uri public key template`
+
+The URI template for the chain's beacon chain explorer public keys. This will be used to generate public key links to the beacon chain explorer. For example: `https://beaconscan.com/validator/{{publicKey}}`.
+
+### `Currency name`
+
+The name of the chain's native currency.
+
+### `Currency symbol`
+
+The symbol of the chain's native currency.
+
+### `Currency decimals`
+
+The number of decimals of the chain's native currency.
+
+### `Currency logo uri`
+
+The URI of the chain's native currency logo. This will be displayed in the UI.
+
+### `Transaction service uri`
+
+The URI of the chain's [Transaction Service](/core-api/api-safe-transaction-service). This will be used to fetch transaction data from the chain. You will need to deploy one Transaction Service per chain, as well as one `txs-db`, `amqp` and `celery` instances.
```

Comment:
```suggestion
The URI of the chain's [Transaction Service](/core-api/api-safe-transaction-service). This will be used to fetch transaction data from the chain. You will need to deploy one Transaction Service per chain, as well as `txs-db`, `amqp` and `celery` instances.
```